Source/ThirdParty/GltfPipeline/getAccessorByteStride.js

```diff
@@ -17,6 +17,9 @@ define([
      * @returns {Number} The byte stride of the accessor.
      */
     function getAccessorByteStride(gltf, accessor) {
+        if(defined(accessor.byteStride) && accessor.byteStride !== 0){
+            return accessor.byteStride;
+        }
```

Comment:
Good catch here. This change probably belongs in `updateVersion` instead.

```
var bufferView = clone(bufferViews[oldBufferViewId]);
var accessorByteStride = (accessor.byteStride === 0) ? getAccessorByteStride(gltf, accessor) : accessor.byteStride;
if (defined(accessorByteStride)) {
```

Reply:
I moved the check into updateversion, but i kept the defined check because I am not sure if accessor.byteStride is always defined?

Reply:
Ah good point.